proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "regen/ecocredit/basket/v1/types.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
```

Comment:
```suggestion
option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket";
```

Reply:
This is the same destination as `state.proto`, `tx.proto`, and `types.proto`. I don't think we want to change here unless we want to change each of those as well.

Reply:
Updated per our discssion.